Formula/git-when-merged.rb

```diff
@@ -1,20 +1,29 @@
 class GitWhenMerged < Formula
+  include Language::Python::Shebang
+
   desc "Find where a commit was merged in git"
   homepage "https://github.com/mhagger/git-when-merged"
   url "https://github.com/mhagger/git-when-merged/archive/v1.2.0.tar.gz"
   sha256 "3fb3ee2f186103c2dae1e4a2e104bc37199641f4ffdb38d85ca612cf16636982"
-  license "GPL-2.0"
+  license "GPL-2.0-only"
+
+  revision 1
 
   bottle do
     sha256 cellar: :any_skip_relocation, all: "67a1db2415c0aa7026ad1958a6966b909575ded3f7c5c63297aff72ca8eb76b0"
   end
 
+  depends_on "python@3.10"
```

Comment:
Could this use:
```suggestion
  uses_from_macos "python"
```

Reply:
thanks, fixed.

Reply:
oops, got error: `ShebangDetectionError: Cannot detect Python shebang: formula does not depend on Python.`

Reply:
@SMillerDev I have rolled back to the inicial version, is it ok?